### PR TITLE
[CUBRIDMAN-198] [EN] Manual for CBRD-24737 (warning message)

### PR DESCRIPTION
**Pull request by hornetmj**
http://jira.cubrid.org/browse/CUBRIDMAN-198

add warning message about loaddb -C. (english manual)

### CODE REVIEW

File: en/admin/migration.inc [hunk 1]
```diff
@@ -551,6 +551,10 @@ The following table shows [options] available with the **cubrid loaddb** utility
 
     The **\-\-no-logging** option enables to load data file quickly when **loaddb** is executed by not storing transaction logs; however, it has risk, which data cannot be recovered in case of errors occurred such as incorrect file format or system failure. In this case, you must rebuild database to solve the problem. Thus, in general, it is not recommended to use this option exception of building a new database which does not require data recovery. If you use this option, **loaddb** does not check the errors like unique violation. To use this option, you should consider these issues.
 
+.. warning::
+
+    When running **loaddb**, foreign key constraints are not checked to ensure performance for loading large amounts of data and to minimize dependencies between loaded data. So, the data inconsistencies with the master node may occur because foreign key constraints are checked during the replication process of the slave or replica node in the HA environment if the **-C** option is used.
```
Comment (kisoo-han) [inline en/admin/migration.inc:556]: When running **loaddb**, foreign key constraints **are not**

**suggest**

 When running **loaddb**, foreign key constraints **will not be**

**Review by kisoo-han — APPROVED** [review]
it trivial

File: en/admin/migration.inc (hunk 1 above)
Comment (mhoh3963) [reply en/admin/migration.inc:556]: suggestion.
if the **-C** option is used.
-> if the loaddb with -C option is executed in the master.

**Review by sungc-cubrid — APPROVED** [review]

**Review by ssihil — APPROVED** [review]

File: en/admin/migration.inc (hunk 1 above)
Comment (hornetmj) [reply en/admin/migration.inc:556]: @kisoo-han Is that expression more grammatically correct?

Comment (hornetmj) [reply en/admin/migration.inc:556]: > suggestion. if the **-C** option is used. -> if the loaddb with -C option is executed in the master.

I think that the sentence already include the meaning of 'is executed in the master'. Nevertheless, do you want fix it?

Comment (mhoh3963) [reply en/admin/migration.inc:556]: @hornetmj 
It's up to you.

**Review by mhoh3963 — APPROVED** [review]

**Pull request merged**